files/build_templates/init_cfg.json.j2

```diff
@@ -18,17 +18,25 @@
         }
     },
     "FEATURE": {
-{%- for feature, status in [("sflow", "disabled"), ("telemetry", "enabled")] %}
+{%- for feature, state, autorestart in [("bgp", "enabled", "enabled"),
+                            ("dhcp_relay", "enabled", "enabled"),
+                            {% if include_iccpd == "y" %}("iccpd", "disabled", "enabled"),{% endif %}
+                            ("lldp", "enabled", "enabled"),
+                            {% if include_mgmt_framework == "y" %}("mgmt-framework", "enabled", "enabled"),{% endif %}
+                            {% if include_nat == "y" %}("nat", "disabled", "enabled"),{% endif %}
+                            ("pmon", "enabled", "enabled"),
+                            ("radv", "enabled", "enabled"),
+                            {% if include_restapi == "y" %}("restapi", "disabled", "enabled"),{% endif %}
+                            {% if include_sflow == "y" %}("sflow", "disabled", "enabled"),{% endif %}
+                            ("snmp", "enabled", "enabled"),
+                            ("swss", "enabled", "enabled"),
+                            ("syncd", "enabled", "enabled"),
+                            ("teamd", "enabled", "enabled"),
+                            {% if include_system_telemetry == "y" %}("telemetry", "enabled", "enabled"),{% endif %}
+                            ("database", "enabled", "disabled")] %}
```

Comment:
Suggest to indent these lines further to align with `("bgp", "enabled", "enabled")`, *or* move `("bgp", "enabled", "enabled")` to the next line and decrease indent, a la,

```
{%- for feature, state, autorestart in [
        ("bgp", "enabled", "enabled"),
        ("dhcp_relay", "enabled", "enabled"),
        {% if include_iccpd == "y" %}("iccpd", "disabled", "enabled"),{% endif %}
        ...
        ("database", "enabled", "disabled")] %}
```